backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3725,4 +3725,70 @@ databaseChangeLog:
             GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
       rollback:
         sql: |
-          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
\ No newline at end of file
+          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
+  - changeSet:
+      id: create-upload-table
+      author: dsass@skylight.digital
+      comment: Add a table that holds data about bulk test result uploads.
+      changes:
+        - tagDatabase:
+            tag: create-upload-table
+        - sql:
+            remarks: Create the enumerations for upload statuses
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.UPLOAD_STATUS as ENUM('SUCCESS', 'FAILURE', 'PENDING');
+        - createTable:
+            tableName: upload
+            remarks: A collection of meta data for bulk uploads.
+            columns:
+              - column:
+                  name: report_id
+                  type: uuid
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column: *created_at_column
+              - column: *created_by_column
+              - column:
+                  name: records_count
+                  remarks: The number of records uploaded.
+                  type: int
+                  constraints:
+                    nullable: false
+              - column:
+                  name: status
+                  remarks: The status of the upload.
+                  type: ${database.defaultSchemaName}.UPLOAD_STATUS
+                  constraints:
+                    nullable: false
+              - column:
+                  name: org_id
+                  remarks: The organization id that uploaded the data.
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__upload__organization
+                    references: organization(internal_id)
+              - column:
+                  name: facility_id
+                  remarks: The facility id that uploaded the data.
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__upload__facility
+                    references: facility(internal_id)
+              - column:
+                  name: errors
+                  remarks: Any errors encountered when uploading.
+                  type: text
+              - column:
+                  name: warnings
+                  remarks: Any warnings encountered when uploading.
+                  type: text
```

Comment:
A couple of things I'm wondering here:

- Are the `errors` and `warnings` we get back actually going to be text? ReportStream seems to send them as arrays of objects ([sample response](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/examples/submission/example1-async-response.json)) which would complicate things
- I wonder if we would ever need to worry about PII in these columns. Based on the response linked above, it probably returns the values that produces an error/warning, which could conceivably be a name/address/etc. I don't feel _super_ worried about this, but I did want to flag it in case anyone else has thoughts
    - To be clear, PII in the database is obviously OK, we just wouldn't grant access to those columns for the Metabase user

Reply:
two very good callouts

Reply:
It looks like errors is an array of json - I'd wager warnings is the same.
here is an example error response I got from stg

```  "errors" : [ {
    "scope" : "report",
    "message" : "CSV file has an inconsistent number of columns on row: 6"
  } ],
```

Reply:
here is a reference to these two fields
https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/examples/submission/example1-async-response.json

Reply:
confirmed that no PII should be coming back in the response and we should be good